packages/@react-spectrum/card/src/CardView.tsx

```diff
@@ -107,21 +101,32 @@ function CardView<T extends object>(props: SpectrumCardViewProps<T>, ref: DOMRef
     focusedKey = focusedItem.parentKey;
   }
 
+  let margin = cardViewLayout.margin || 0;
+  let virtualizer = cardViewLayout.virtualizer;
+  let scrollToItem = useCallback((key) => {
+    virtualizer && virtualizer.scrollToItem(key, {
+      duration: 0,
+      offsetY: margin
+    });
+  }, [margin, virtualizer]);
```

Comment:
Makes it so the entire Card is brought into view when the user focuses it via arrow keys navigation. Previously the focus ring was cut off since it is a outset focus ring